src/fusor/harvester.py

```diff
@@ -45,6 +45,8 @@
 T = TypeVar("T", bound=Translator)
 
 
+# NOTE: the pydantic dataclass has errors validating the `annotated` field.
+# We didn't fully investigate why, but the stdlib dataclass doesn't have this issue, so using it instead.
```

Comment:
@jarbesfeld I was able to resolve this issue quickly. I'd like for you to try resolving this on your own first. Please revert back to using Pydantic dataclass so we can have validation.